README.md

```diff
@@ -25,8 +25,6 @@ sudo apt-get install nodejs cmake build-essential swig python3.8
 sudo npm install --global npm
```

Comment:
Using a variable in the "=" block fails with output:
> Invalid operand type

<img width="184" alt="image" src="https://user-images.githubusercontent.com/126523/221427179-84801333-e49d-4061-8c06-05e32d5590dd.png">

```xml
<xml xmlns="http://www.w3.org/1999/xhtml">
  <variables>
    <variable type="" id=";/fOvI=r,rhuvz3kg:!y" islocal="false" iscloud="false">i</variable>
    <variable type="" id="mXKmvt*Z7vJ|3:~+HU:#" islocal="false" iscloud="false">j</variable>
    <variable type="" id="2ogUd*aq6_gfGWacz#/2" islocal="false" iscloud="false">testvar</variable>
    <variable type="" id="q=qhREa7eU=jdV,%uUs-" islocal="false" iscloud="false">k</variable>
    <variable type="" id="m+BqD().ru``m17|I[)," islocal="false" iscloud="false">m</variable>
  </variables>
  <block type="control_run" id="0fNAs-vY8^BL;[T$]eav" x="159" y="8">
    <next>
      <block type="data_setvariableto" id="(RpP{VpAeA!KNbm}Tscv">
        <field name="VARIABLE" id="2ogUd*aq6_gfGWacz#/2" variabletype="">testvar</field>
        <value name="VALUE">
          <shadow type="text" id="2#bAQ,?kgq|Jf~}kEU$P">
            <field name="TEXT">4</field>
          </shadow>
        </value>
        <next>
          <block type="control_if" id="eU;`eR~$4=MUK7b9sG0E">
            <value name="CONDITION">
              <block type="operator_equals" id="=%$4Pc2{Ju]u:ag%L37H">
                <value name="OPERAND1">
                  <shadow type="text" id="zNi~dy7NAcJNT~G-;gx8">
                    <field name="TEXT">1</field>
                  </shadow>
                  <block type="data_variable" id="BCKI$6kiG|4c/|}Ob2ul">
                    <field name="VARIABLE" id="2ogUd*aq6_gfGWacz#/2" variabletype="">testvar</field>
                  </block>
                </value>
                <value name="OPERAND2">
                  <shadow type="text" id="OUt;O2{3..-Mk6*]LPCe">
                    <field name="TEXT">1</field>
                  </shadow>
                </value>
              </block>
            </value>
          </block>
        </next>
      </block>
    </next>
  </block>
</xml>
```